Source/HYPScatterPlot.m

```diff
@@ -34,6 +34,10 @@ @interface HYPScatterPlot()
 
 @property (nonatomic) UIView *plotView;
 
+@property (nonatomic) UILongPressGestureRecognizer *panGestureRecognizer;
```

Comment:
panGestureRecognizer != longPressGestureRecognizer

You might want to reconsider the naming of this variable